website/mod.ts

```diff
@@ -41,7 +41,7 @@ export interface Props {
   routes?: Routes[];
 
   /** @title Seo */
-  seo?: Omit<Seo, "jsonLDs">;
+  seo?: Omit<Seo, "jsonLDs" | "titleTemplate" | "descriptionTemplate" | "canonical">;
```

Comment:
Wouldn't `Pick<Seo, whatever` make more sense here?

Reply:
After your comment I tested it, but it did not work with Pick, @mcandeia is it mapped?

Anyway, I will merge to not loose the improvement.